# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/ServiceWorkerRegistrationProxy.h
# Commit: 2e24d997d5de
# Full Hash: 2e24d997d5de727be8e44b522e57d7ff7db52486
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-07-16 21:45:35
# Regressor Bug: 1972117
# File Overlap Count: 1
# Description:
#   Revert "Bug 1972117: apply code formatting via Lando" for causing assertion failures @ MozPromise.h
#   
#   This reverts commit e76333b4d7fb2d323e797f12bde50ccc99a51a85.
#   
#   Revert "Bug 1972117 - Part 4: Clear notifications on service worker unregistration r=asuth"
# ==============================================================================

diff -r c257659aa2b4 -r 2e24d997d5de dom/serviceworkers/ServiceWorkerRegistrationProxy.h
--- a/dom/serviceworkers/ServiceWorkerRegistrationProxy.h	Wed Jul 16 19:15:43 2025 +0300
+++ b/dom/serviceworkers/ServiceWorkerRegistrationProxy.h	Wed Jul 16 19:18:03 2025 +0300
@@ -9,7 +9,6 @@
 
 #include "mozilla/dom/ClientInfo.h"
 #include "mozilla/dom/PServiceWorkerRegistrationParent.h"
-#include "mozilla/dom/notification/NotificationUtils.h"
 #include "nsProxyRelease.h"
 #include "ServiceWorkerRegistrationDescriptor.h"
 #include "ServiceWorkerRegistrationListener.h"
@@ -87,8 +86,7 @@
 
   RefPtr<NavigationPreloadStatePromise> GetNavigationPreloadState();
 
-  RefPtr<notification::NotificationsPromise> GetNotifications(
-      const nsAString& aTag);
+  RefPtr<NotificationsPromise> GetNotifications(const nsAString& aTag);
 
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(ServiceWorkerRegistrationProxy,
                                         override);